### PR TITLE
Update model cards

### PR DESCRIPTION
**Pull request by AkshitaB**
* Added `short_description` for a one-line summary for the model.
* Miscellaneous information updates for all model cards.

### CODE REVIEW

File: tests/pretrained_test.py [hunk 1]
```diff
@@ -406,8 +406,11 @@ def test_transformer_qa(self):
 
     @pytest.mark.parametrize("model_id, model_card", get_pretrained_models().items())
     def test_pretrained_models(self, model_id, model_card):
-        # Each model in pretrained_models should have an archive and registered_predictor_name.
+        # Each model in pretrained_models should have an archive and a description.
         assert model_card.archive_file is not None
+        assert model_card.display_name is not None
```
Comment (jonborchardt) [inline tests/pretrained_test.py:411]: we should add id to this list

**Review by jonborchardt — APPROVED** [review]
looks fine to me, but someone else should take a look at the actual values....

**Review by epwalsh — APPROVED** [review]
LGTM!

File: tests/pretrained_test.py (hunk 1 above)
Comment (AkshitaB) [reply tests/pretrained_test.py:411]: A model card cannot be instantiated without `id`. So, no need to check it again.

**Pull request merged**